azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/MainActivity.kt

```diff
@@ -130,25 +127,6 @@ internal class MainActivity : AppCompatActivity() {
         permissionManager.setAudioPermissionsState()
     }
 
-    private fun injectDependencies() {
```

Comment:
These fields now have getters that reference the container directly.

We don't need to set up the holder, since the holder is self-initializing now, assuming it has a instanceID, container will create when you ask for it.